apps/token-manager/contracts/TokenManager.sol

```diff
@@ -38,8 +38,8 @@ contract TokenManager is ITokenController, AragonApp, IForwarder {
         bool revokable;
     }
 
-    mapping (address => TokenVesting[]) vestings;
-    mapping (address => bool) everHeld;
+    mapping (address => TokenVesting[]) internal vestings;
+    mapping (address => bool) internal everHeld;
```

Comment:
I guess it's not a big deal (there is `allHolders`) but I don't see a getter for this one.

Reply:
Good point, this can be public :).